README.md

```diff
@@ -6,7 +6,23 @@ Release automation for pandas.
   - [  ] `TAG` in `Makefile`
   - [  ] `GH_USERNAME` in `Makefile`
 
-If running for the first time be sure to initialize repos
+## **If running for the first time:**
+
+- Create a conda enviroment based on environment.yml
+```sh
+conda env create -f environment.yml
+```
+- Activate the environment 
+```sh
+conda activate pandas-release
+```
+- Be sure you have the following forked on your github
+  - https://github.com/pandas-dev/pandas-website
+  - https://github.com/conda-forge/pandas-feedstock
+  - https://github.com/MacPython/pandas-wheels
+  - https://github.com/pandas-dev/pandas
+
+- Be sure to initialize repos
```

Comment:
Small thing, but may be a colon at the end to show that you're talking about the next command (thought initially you were talking about the previous list of repos)